google/cloud/dataproc/README.md

```diff
@@ -3,7 +3,11 @@
 :construction:
 
 This directory contains an idiomatic C++ client library for the
-[Cloud Dataproc API][cloud-service-docs], a service to Manages Hadoop-based clusters and jobs on Google Cloud Platform.
+[Cloud Dataproc API][cloud-service-docs], a fully managed and highly scalable
+service for running Apache Spark, Apache Flink, Presto, and 30+ open source
+tools and frameworks. Use Dataproc for data lake modernization, ETL, and secure
+data science, at planet scale, fully integrated with Google Cloud, at a fraction
+of the cost.
 
```

Comment:
I think this description may give the impression that you can run C++ applications in Cloud Dataproc, which I think you cannot?  Maybe we should say something like: "Use this library to manage Cloud Dataproc clusters and other resources, to run C++ applications in Cloud Dataproc we recommend you use ..." (if we have a thing to fill the blank), or say "Note that this library allows you to *manage* Cloud Dataproc resources, it does not provide APIs to run C++ applications in Cloud Dataproc".